andela/ah-orcas
# #161254669 : favorite and unfavorite an article

#### What does this PR do?
```Adds functionality for users to favorite or unfavorite an article  ```
#### Description of Task to be completed?
```
an authenticated user hitting api/articles/<slug>/favorite/
with POST request should be able to favorite an article
with DELETE request should be able to unfavorite an article
 ```
#### How should this be manually tested?
$ virtualenv env
$ source env/bin/activate
$ cd env
```
- Clone this repo.
``` 
$ git clone https://github.com/andela/ah-orcas.git
```
- Switch to this branch
``` 
$  git checkout  ft-favorite-article-161254669
```
- Create a .env file and run it. Check sample.env file to configure yours.
```
$ source .env
```
- Run the app
```
$ python manage.py runserver
#### Any background context you want to provide?
```Authentication is required```
#### What are the relevant pivotal tracker stories?

[#161254669](https://www.pivotaltracker.com/story/show/161254669)
